spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/data/TestHelpers.java

```diff
@@ -784,11 +782,6 @@ private static void assertEquals(
     for (int i = 0; i < actual.numFields(); i += 1) {
       StructField field = struct.fields()[i];
       DataType type = field.dataType();
-      // ColumnarRow.get doesn't support TimestampNTZType, causing tests to fail. the representation
```

Comment:
thanks for removing this. This also fixes https://github.com/apache/iceberg/issues/11945